index.html

```diff
@@ -0,0 +1,541 @@
+
+
+        </p>
+        <p class="content">
+            What would become our observable universe mushroomed in size at least a trillion trillionfold — from a submicroscopic speck of primordial energy to the size of a grapefruit — in less than a cosmic eye-blink.
+
+
+        </p>
+        <p class="content">
+            Almost as quickly, this pulse would subside, relaxing into ordinary particles and radiation. All of normal cosmic history was still ahead, resulting in today’s observable universe, a patch of sky and stars billions of light-years across. “It’s often said that there is no such thing as a free lunch,” Dr. Guth likes to say, “but the universe might be the ultimate free lunch.”
+        </p>
+        <p class="content">
+            Make that free lunches. Most of the hundred or so models resulting from Dr. Guth’s original vision suggest that inflation, once started, is eternal. Even as our own universe settled down to a comfortable homey expansion, the rest of the cosmos will continue blowing up, spinning off other bubbles endlessly, a concept known as the multiverse.
+        </p>
+        <p class="content">
+            So the future of the cosmos is perhaps bright and fecund, but do not bother asking about going any deeper into the past.
+        </p>
+        <p class="content">
+            We might never know what happened before inflation, at the very beginning, because inflation erases everything that came before it. All the chaos and randomness of the primordial moment are swept away, forever out of our view.
+        </p>
+        <p class="content">
+            “If you trace your cosmic roots,” said Abraham Loeb, a Harvard-Smithsonian astronomer who was not part of the team, “you wind up at inflation.”
+        </p>
+        <p class="content">
+            <small class="small">A version of this article appears in print on March 18, 2014, Section A, Page 1 of the New York edition with the headline: Space Ripples Reveal Big Bang’s Smoking Gun. <a href="#">Order Reprints</a> | <a href="#">Today’s Paper</a> | <a href="#">Subscribe</a></small>             
+        </p>
+        </article>
+        <div class="bottom-social flex">
+        <button class="comments-btn" type= "button"> READ 615 COMMENTS</button>
+            <div class="social-media-bottom flex">
+                <a href="#"><i class="fab fa-facebook-f"></i></a>
+                <a href="#"><i class="fab fa-whatsapp"></i></a>
+                <a href="#"><i class="fab fa-twitter"></i></a>
+                <a href="#"><i class="fas fa-envelope"></i></a>
+                <a href="#"><i class="fas fa-share"></i></a>
+            </div>
+        </div>
+        <br><br>
+        <!-- suggested newletter section -->
+        <h4 class="suggested-header">Suggested newsetter for you</h4>
```

Comment:
## Mandatory

- [ ] As stated in the previous review, this section should be centered with the appropriate space. Currently, your page's layout is very much close to the edge of the browser.

#### main page 👇 
![image](https://user-images.githubusercontent.com/64360974/113365406-d97d2b80-935e-11eb-9908-44d08489c4cf.png)

#### yours 👇 
![image](https://user-images.githubusercontent.com/64360974/113365729-c28b0900-935f-11eb-873b-68f1808a9166.png)

## optional 

- [ ] Add the plus icons to the top right corner of the images here